test/runtests.jl

```diff
@@ -23,6 +22,10 @@ end
 
 include("TestFunctions.jl")
 
+for target in readdir("util")
+    include("util/$target")
+end
+
```

Comment:
To clarify, I mean moving the test sources to e.g. `test/util/indexing/runtests.jl`. Though you could just as well do away with the `util` subfolder, if you wanted (I'm +0 on that).